src/Game/Game.h

```diff
@@ -110,10 +117,11 @@ class Game
     bool _quit = false;
     SDL_Event _event;
     bool _initialized = false;
+    std::unique_ptr<InputController> inputController_;
```

Comment:
Since there is no problem with undescore in class members i see no reason to change code style it this point